docs/fhir-web-docker-deployment.md

```diff
@@ -0,0 +1,215 @@
+        # - 'EXPRESS_REDIS_SENTINEL_CONFIG={"name":"sentinelMasterName","sentinelPassword":"sentinelMasterPassword","sentinels":[{"host":"sentinel-node-1-domain","port":"12345"},{"host":"sentinel-node-2-domain","port":"12345"},{"host":"sentinel-node-3-domain","port":"12345"}]}'
+    ```
+
+    ```js
+    // config.js.tpl
+
+    window._env_ = {
+      // keycloak
+      REACT_APP_KEYCLOAK_API_BASE_URL: '<keycloak-domain>/auth/admin/realms/<keycloak-realm>',
+      REACT_APP_KEYCLOAK_LOGOUT_URL:
+        '<keycloak-domain>/auth/realms/<keycloak-realm>/protocol/openid-connect/logout',
+      REACT_APP_OPENSRP_ACCESS_TOKEN_URL:
+        '<keycloak-domain>/auth/realms/<keycloak-realm>/protocol/openid-connect/token',
+      REACT_APP_OPENSRP_AUTHORIZATION_URL:
+        '<keycloak-domain>/auth/realms/<keycloak-realm>/protocol/openid-connect/auth',
+      REACT_APP_OPENSRP_CLIENT_ID: '<keycloak-client-id>',
+      REACT_APP_OPENSRP_USER_URL:
+        '<keycloak-domain>/auth/realms/<keycloak-realm>/protocol/openid-connect/userinfo',
+
+      // fhir-web
+      REACT_APP_WEBSITE_NAME: '<website-name>',
+      REACT_APP_OPENSRP_WEB_VERSION: '<fhir-web-tag>',
+      REACT_APP_DOMAIN_NAME: '<fhir-web-domain>',
+      REACT_APP_EXPRESS_OAUTH_GET_STATE_URL: '<fhir-web-domain>/oauth/state',
+      REACT_APP_EXPRESS_OAUTH_LOGOUT_URL: '<fhir-web-domain>/logout',
+
+      // fhir-server
+      REACT_APP_FHIR_API_BASE_URL: '<fhir-server-domain>/fhir',
+
+      // UUID's
+      REACT_APP_FHIR_ROOT_LOCATION_IDENTIFIER: '<randomly-generated-uuid>',
+      REACT_APP_COMMODITIES_LIST_RESOURCE_ID: '<randomly-generated-uuid>',
+
+      // TODO: are these needed for fhir?
+      REACT_APP_DEFAULT_PLAN_ID: '<default-opensrp-plan-uuid>',
+      REACT_APP_ACTION_UUID_NAMESPACE: '<randomly-generated-uuid>',
+      REACT_APP_PLAN_UUID_NAMESPACE: '<randomly-generated-uuid>',
+
+      // toggle fhir-web modules
+      REACT_APP_ENABLE_FHIR_CARE_TEAM: 'false',
+      REACT_APP_ENABLE_FHIR_GROUP: 'false',
+      REACT_APP_ENABLE_FHIR_HEALTHCARE_SERVICES: 'false',
+      REACT_APP_ENABLE_FHIR_LOCATIONS: 'false',
+      REACT_APP_ENABLE_FHIR_PATIENTS: 'false',
+      REACT_APP_ENABLE_FHIR_TEAMS: 'false',
+      REACT_APP_ENABLE_FHIR_USER_MANAGEMENT: 'false',
+      REACT_APP_ENABLE_FHIR_COMMODITY: 'false',
+      REACT_APP_ENABLE_QUEST: 'false',
+
+      // composite fhir-web modules
+      REACT_APP_ENABLE_USER_MANAGEMENT: 'false',
+      REACT_APP_ENABLE_LOCATIONS: 'false',
+      REACT_APP_ENABLE_TEAMS: 'false',
+      REACT_APP_ENABLE_TEAMS_ASSIGNMENT_MODULE: 'false',
+
+      // TODO: are these needed for fhir?
+      REACT_APP_ENABLE_CARD_SUPPORT: 'false',
+      REACT_APP_ENABLE_INVENTORY: 'false',
+      REACT_APP_ENABLE_PLANS: 'false',
+      REACT_APP_ENABLE_PRODUCT_CATALOGUE: 'false',
+      REACT_APP_ENABLE_REPORTS: 'false',
+      REACT_APP_ENABLE_SERVER_SETTINGS: 'false',
+      REACT_APP_ENABLE_FORM_CONFIGURATION: 'false',
```

Comment:
yeah, no this are not fhir modules.

Reply:
@p-netm how about these:

```      
REACT_APP_DEFAULT_PLAN_ID: '<default-opensrp-plan-uuid>',
REACT_APP_ACTION_UUID_NAMESPACE: '<randomly-generated-uuid>',
REACT_APP_PLAN_UUID_NAMESPACE: '<randomly-generated-uuid>',
```

Reply:
Nvm, removed the rest apart form `REACT_APP_DEFAULT_PLAN_ID`